### ISSUE TITLE
tokenIconUrl: add ability to override for test networks

### ISSUE THREAD

**Issue by sohkai** [aragon/aragon-ui#717]
It would be nice to have a way to configure `tokenIconUrl()` so that you can apply a list of overrides if you want to.

For example, how this is currently done in Finance: https://github.com/aragon/aragon-apps/blob/master/apps/finance/app/src/lib/icon-utils.js#L7.

**Comment by bpierre** [comment]
We should probably move this issue to the [useToken()](https://github.com/aragon/use-token) now. I am wondering if it’s something that we should provide as part of the hook API, rather than letting people use a wrapper function when they need this?

**Comment by sohkai** [comment]
Yes, agreed! I can see people writing their own hook around `useToken()` to specify the overrides for this! Or maybe we can move it up as an optional context provider to centralize it in one place?
